pages.de/common/az-logout.md

```diff
@@ -0,0 +1,13 @@
+# az logout
+
+> Melden Sie sich ab, um den Zugriff auf Azure-Abonnements zu entfernen.
+> Teil der `Azure CLI`, der Befehlszeilenschnittstelle von Azure.
```

Comment:
```suggestion
> Teil von `azure-cli`.
```

Reply:
@sidyes What exactly is your rationale for marking this as resolved? You merged basically the same suggestion in #9171, so please at least explain why you disagree here.

Reply:
Sorry that was a mistake